java/java.lsp.server/nbcode/integration/src/org/netbeans/modules/nbcode/integration/layer.xml

```diff
@@ -21,6 +21,11 @@
 -->
 <!DOCTYPE filesystem PUBLIC "-//NetBeans//DTD Filesystem 1.2//EN" "http://www.netbeans.org/dtds/filesystem-1_2.dtd">
 <filesystem>
+    <folder name="Explorers">
+        <folder name="_all">
+            <file name="standard-cookies.contextValues" url="standard-cookies.contextValues"/>
```

Comment:
Interesting API. Is it documented somewhere? I'd rather use `<attr ..../>` than content of a file, however.

Reply:
Not documented yet, and I'd wait with publication for the next few PRs - I am going to play with DB explorer nodes, and probably provide some project-related actions. I'd expect some changes.